core/src/main/scala/chisel3/package.scala

```diff
@@ -420,6 +420,8 @@ package object chisel3 {
     def toTarget:         ReferenceTarget
     def toAbsoluteTarget: ReferenceTarget
     def toRelativeTarget(root: Option[BaseModule]): ReferenceTarget
+
+    def suggestName(name: String): Unit
```

Comment:
Why does this get a blank line but nothing above it does? A ScalaDoc would be good though 🙂